assets/pages/contributions-landing/new_components/ContributionHeader.jsx

```diff
@@ -16,7 +16,7 @@ import SvgRoundel from 'components/svgs/roundel';
 // ----- Types ----- //
 
 type PropTypes = {
-  selectedCountryGroup: CountryGroup,
+  selectedCountryGroup: ?CountryGroup,
```

Comment:
Is this an option now because the component is shared across views where the user cannot change their country group?

Reply:
yes